types/react-image-crop/index.d.ts

```diff
@@ -67,6 +60,7 @@ declare class ReactCrop extends Component<ReactCrop.ReactCropProps> {
         width: number,
         height: number,
     };
+    resolveCropAndTriggerChange: (crop: ReactCrop.Crop, image: HTMLImageElement) => ReactCrop.Crop;
```

Comment:
I think this should just be `resolveCrop`:
https://github.com/DominicTobias/react-image-crop/blob/master/lib/ReactCrop.js#L101

Reply:
Of course!